src/foam/nanos/auth/UserAndGroupAuthService.java

```diff
@@ -35,6 +35,13 @@
   // pattern used to check if password has only alphanumeric characters
   java.util.regex.Pattern alphanumeric = java.util.regex.Pattern.compile("[^a-zA-Z0-9]");
 
+  public UserAndGroupAuthService(X x) {
+    setX(x);
```

Comment:
Do we need this?  Couldn't we just proxy the start() method in our PasswordExpireAuthService?